### PR TITLE
 Adding new PHP extensions

### PR DESCRIPTION
**Pull request by G-Arvind**
## Purpose of this pull request

This pull request covers the process of adding a new PHP extension to PHP images of Cloud Docker.
(https://github.com/magento/magento-cloud-docker/issues/155)

## Affected DevDocs pages

-  https://devdocs.magento.com/cloud/docker/docker-extend.html

whats new
Updated the [Extend Docker](https://devdocs.magento.com/cloud/docker/docker-extend.html)  topic to include instructions for adding a new PHP extension to the PHP container for Magento Cloud Docker.

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/cloud/docker/docker-extend.md [hunk 1]
```diff
@@ -75,6 +75,60 @@ To add a new service version using a `Dockerfile`:
...
+        ```
+   * Add the .ini files inside `etc` directory
+   * Open `Dockerfile` and add the following line to it
+
+        ```COPY etc/<filename>.ini /usr/local/etc/php/conf.d/<filename>.ini```
+   * For CLI container, Navigate to the following directory
+
+        ```bash
+        cd magento-cloud-docker/images/php/cli
+        ```
```
Comment (meker12) [inline src/cloud/docker/docker-extend.md:119]:
```suggestion
        ```conf
        COPY etc/<filename>.ini /usr/local/etc/php/conf.d/<filename>.ini
        ```
   -  For the CLI container, navigate to the following directory

        ```bash
        cd magento-cloud-docker/images/php/cli
        ```
```

File: src/cloud/docker/docker-extend.md [hunk 2]
```diff
@@ -75,6 +75,60 @@ To add a new service version using a `Dockerfile`:
...
+   * Add the .ini files inside `etc` directory
+   * Open `Dockerfile` and add the following line to it
+
+      ```ADD etc/<file-name>.ini /usr/local/etc/php/conf.d/<filename>.ini```
+
```
Comment (meker12) [inline src/cloud/docker/docker-extend.md:124]:
```suggestion
   -  Open `Dockerfile` and add the following line to it

      ```conf
      ADD etc/<file-name>.ini /usr/local/etc/php/conf.d/<filename>.ini
      ```

```

Comment (G-Arvind) [reply src/cloud/docker/docker-extend.md]: Hi @meker12 , thanks a lot for the suggestions, I have made all the changes and fixed the spacing issues.
Thanks.

**Comment by meker12** [comment]
@G-Arvind  Merged your changes in https://github.com/magento/devdocs/pull/7269.

Thanks very much for your contribution!

**Pull request closed without merging**